dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpBaseMessageImpl.java

```diff
@@ -2924,10 +2924,12 @@ else if (getServiceContext().getHttpConfig().onlySameSiteStar()) {
                     // Set Partitioned Flag for SameSite=None Cookie
                     if (getServiceContext().getHttpConfig().getPartitioned() == true
                         && sameSiteAttributeValue.equalsIgnoreCase(HttpConfigConstants.SameSite.NONE.getName())) {
-                        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-                            Tr.debug(tc, "[1] Setting the Partitioned attribute for SameSite=None");
-                        }
-                        cookie.setAttribute("partitioned", "");
+                            if(cookie.getAttribute("partitioned") == null) {  // null means no value has been set yet
```

Comment:
Any need for beta checks here?

Reply:
This check is nested within the condition: "getServiceContext().getHttpConfig().getPartitioned() == true" (httpEndpoint samesite config) which is internal and beta guarded. 

I can add a check if you prefer?

Reply:
if we're eventually guarded then I'm fine with it as is.